gcloud-java-bigquery/src/main/java/com/google/gcloud/spi/BigQueryRpc.java

```diff
@@ -185,4 +186,26 @@ GetQueryResultsResponse getQueryResults(String jobId, Map<Option, ?> options)
       throws BigQueryException;
 
   QueryResponse query(QueryRequest request) throws BigQueryException;
+
+  /**
+   * Opens a resumable upload session to load data into a BigQuery table and returns an upload URI.
+   *
+   * @param configuration load configuration
+   * @throws BigQueryException upon failure
+   */
+  String open(JobConfigurationLoad configuration) throws BigQueryException;
+
+  /**
+   * Uploads the provided data to the resumable upload session at the specified position.
+   *
+   * @param uploadId the resumable upload session URI
+   * @param toWrite a byte array of data to upload
+   * @param toWriteOffset offset in the {@code toWrite} param to start writing from
+   * @param destOffset offset in the destition where to upload data to
+   * @param length the number of butes to upload
+   * @param last if {@code true} indicates that the last chunk if being uploaded
```

Comment:
The second "if" (near the end of the line) should be an "is".  This also might be clearer if worded as "true indicates that the ..." (removing the "if")